opm/input/eclipse/Schedule/CompletedCells.hpp

```diff
@@ -89,28 +189,70 @@ class CompletedCells
             serializer(this->depth);
             serializer(this->dimensions);
         }
-
-        Cell(std::size_t g, std::size_t i_, std::size_t j_, std::size_t k_)
-            : global_index(g)
-            , i(i_)
-            , j(j_)
-            , k(k_)
-        {}
-
-        Cell() = default;
     };
 
+    /// Default constructor.
+    ///
+    /// Creates a collection that is only usable as the target of a
+    /// deserialisation operation.
     CompletedCells() = default;
-    ~CompletedCells() = default;
+
+    /// Constructor.
+    ///
+    /// \param[in] dims Host grid's Cartesian dimensions.  Needed to
+    /// translate between linearlised Cartesian indices and (I,J,K) tuples.
```

Comment:
linearised

Reply:
> linearised

Thanks.  I've pushed an update to address that.